test/test_pattern_matcher.py

```diff
@@ -127,6 +129,117 @@ def test_allow_len(self):
     self.assertEqual(matcher.rewrite(c6), None)
     self.assertEqual(matcher.rewrite(c7), c7)
 
+  def test_constant_arg(self):
```

Comment:
redundant test, we already have `test_arg`

Reply:
removed test